[SOAR 1789] Bug fixes and stability improvements for Block Host with FortiGate workflows (#266)
rapid7/insightconnect-workflows: workflows/Block_Host_with_Fortinet_Firewall_from_Slack/help.md

# Version History

> * 1.1.0 - Update workflow to handle block requests for address that already exist | Update workflow to continue on failure and send messages indicating that something went wrong

Reviewer: ```suggestion
* 1.1.0 - Update workflow to handle block requests for address that already exists | Update workflow to send Slack message when a job failure has occurred
```
